examples/scripts/multi_fitting.py

```diff
@@ -0,0 +1,79 @@
+import numpy as np
+
+import pybop
+
+# Parameter set and model definition
+parameter_set = pybop.ParameterSet.pybamm("Chen2020")
+model_1 = pybop.lithium_ion.SPM(parameter_set=parameter_set)
+model_2 = pybop.lithium_ion.SPM(parameter_set=parameter_set.copy())
```

Comment:
I suspect we should implement a `copy()` method for the model class to simplify this.

Reply:
Right, we have one already: https://github.com/pybop-team/PyBOP/blob/4638a60f1bc00afef0bd2c8e179b7c17826289d4/pybop/models/base_model.py#L582

Reply:
At the moment, copying the model class does not seem to create a new `parameter_set`, which then causes problems if different parameters are later set as inputs. Do you think that `model.copy()` should point to the existing parameter set or create a copy of the parameter set?

Reply:
I added a `model.new_copy()` method to resolve this issue.